src/support/import-supervisor.js

```diff
@@ -174,22 +177,12 @@ function ImportSupervisor(services, config) {
     // Hash the API Key to ensure it is not stored in plain text
     const hashedApiKey = hashWithSHA256(importApiKey);
 
-    let updatedOptions = { ...options };
-
-    if (importScript) {
-      updatedOptions = { ...updatedOptions, hasCustomImportJs: true };
-    }
-
-    if (customHeaders) {
-      updatedOptions = { ...updatedOptions, hasCustomHeaders: true };
-    }
-
```

Comment:
How will we expose these values for consumption both by API clients, as well as for use by our dashboards? cc @swetabar

Reply:
We will have to provide new columns in the job for these values.   We can log an new issue for this.

Reply:
SITES-25634